cli/githubutil.py

```diff
@@ -140,3 +141,101 @@ def release_note_blocks_cli(
         repository_branch=repository_branch,
         commit_range=commit_range
     ).release_note_blocks()
+
+
+def cleanup_draft_releases(
+    github_cfg_name: str,
+    github_repository_owner: str,
```

Comment:
nit: it might be more convenient to merge this into one arg (repo_path) so callers need to type less ;-)